contrib/babelfishpg_tsql/src/pl_exec-2.c

```diff
@@ -1548,12 +1548,12 @@ exec_stmt_exec_batch(PLtsql_execstate *estate, PLtsql_stmt_exec_batch *stmt)
 	PG_FINALLY();
 	{
 		/* Restore past settings */
+		pltsql_revert_guc(save_nestlevel);
+		pltsql_revert_last_scope_identity(scope_level);
+
```

Comment:
Unsure if `set_session_properties` can fail. Seems easier to just revert the nest levels first.